### ISSUE TITLE
Use external player

### ISSUE THREAD

**Issue by KlassenKonstantin** [justintv/Twitch-API#434]
Is it allowed to not use a webview with an embedded iFrame but an external player like Exoplayer for instance?

**Comment by BarryCarlyon** [comment]
I don't believe so as that bypasses the ability to for Twitch to serve videos which I believe is against the TOS.

Refer to https://github.com/justintv/Twitch-API/blob/master/embedding.md for "off site" viewing

**Comment by Sirgue** [comment]
Try Livestreamer
http://docs.livestreamer.io/index.html
http://docs.livestreamer.io/api_guide.html
